perun-base/src/test/resources/test-schema.sql

```diff
@@ -180,17 +180,18 @@ create table groups (
 );
 
 create table members (
-	id integer not null,
-	user_id integer not null,
-	vo_id integer not null,
+	id integer not null PRIMARY KEY,
+	user_id integer not null REFERENCES users(id) ON DELETE CASCADE,
+	vo_id integer not null REFERENCES vos(id) ON DELETE CASCADE,
```

Comment:
We do not use ``ON DELETE CASCADE``, since we want to audit all delete actions and remove all relations in Bl. I believe, it's used only in registrar, for the simplification of actions in core (since core can't reach registrar methods and logic would have to be duplicated).

Reply:
OK, removed